### PR TITLE
Create notAperson535.json

### PR DESCRIPTION
**Pull request by notAperson535**
My about me website

### CODE REVIEW

File: domains/notAperson535.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My about me website",
+  "repo": "https://github.com/notAperson535/notAperson535",
+  "owner": {
+    "username": "notAperson535",
+    "email": "notAperson939@gmail.com",
+  },
+  "record": {
+    "CNAME": "notAperson535.github.io/notAperson535"
```
Comment (MaskDuck) [inline domains/notAperson535.json:9]: cname cant have slash

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by notAperson535** [comment]
@MaskDuck I am trying to put my profile readme as my `.is-a.dev` domain. I saw someone else do it.

**Comment by MaskDuck** [comment]
you cant

this would fail in the ci checks

**Comment by notAperson535** [comment]
> 

@andrewstech how did you put [this](https://github.com/andrewstech/andrewstech) as your github.io page?

**Comment by MaskDuck** [comment]
no it isnt :)

**Comment by andrewstech** [comment]
> > I am trying to put my profile readme as my .is-a.dev domain. I saw someone else do it.
> 
> @andrewstech how did you put [this](https://github.com/andrewstech/andrewstech) as your github.io page?

Rename your repository to 
notAperson535.github.io

Wait for the ci to pass then rename your repository back to notAperson535/notAperson535. 

Done

**Comment by MaskDuck** [comment]
im waiting for someone to approve my domain request tbh

**Comment by MaskDuck** [comment]
@notAperson535 now change cname to `notaperson535.github.io`

**Comment by MaskDuck** [comment]
for some reason ci is not working, lemme close and reopen to see if it fixes

**Review by MaskDuck — APPROVED** [review]
nice

**Comment by MaskDuck** [comment]
@notAperson535 
### [optional] star the repo pls

The changes have been published!! It should reflect in less than 24 hours.

## Here's what you need to do next
If your domain points to a server you own, add `domain-name.is-a.dev` to your server config. For https, you will have to configure ssl certificate to allow the new subdomain.

### For github pages users,
* Go to your github page repo (`user/user.github.io`)
* Open up the **settings** tab
* Scroll down to the **Github pages** section
* In the **Custom domain** text input, enter the domain you registered (`domain-name.is-a.dev`)
* Check the **Enforce HTTPS** checkbox below the input
* Give it some time to reflect and you should be good to go

## Need help with your domain?
If you are having trouble setting up your domain, [create an issue](https://github.com/is-a-dev/register/issues/new/choose). I will try my best to get back to you asap!

## Made a mistake in the record?
Don't worry, you can create a new pull request with the corrections

## Love/Hate the service?
**Love it?** Leave it a **star**! Also consider donating so that I can keep this service running forever!

**Hate it?** Please leave your feedback by [creating an issue](https://github.com/is-a-dev/register/issues/new/choose). I would really like to keep improving this service for other users.

## Wanna support this project?

Help me in my mission to keep this service alive forever by donating!

<a href="https://www.buymeacoffee.com/phenax" target="_blank">Buy Me A Coffee</a> <a href="https://liberapay.com/phenax" target="_blank">Liberapay recurring donation button</a>

**Comment by notAperson535** [comment]
@MaskDuck thanks!

**Comment by notAperson535** [comment]
@MaskDuck I assigned `https://github.com/notAperson535/notAperson535` to notAperson535.is-a.dev but it keeps redirecting to is-a.dev website

**Comment by andrewstech** [comment]
> notAperson535.is-a.dev

Showing for me. 
image

**Comment by notAperson535** [comment]
> > notAperson535.is-a.dev
> 
> Showing for me. image

Strange, it works in chrome but not edge

**Comment by MaskDuck** [comment]
1. showing for me.
2. it wouldnt be nice to ping peoples, ill receive the notification anyway because i modified the open/close stats.
however i am busy waiting for people to approve my domain request why are they so slow :sob:

**Comment by MaskDuck** [comment]
image
can confirm, it does work on brave either

**Comment by notAperson535** [comment]
maybe my Microsoft Edge is out of date.

**Comment by notAperson535** [comment]
I just cleared my recent cookies and it works now

**Pull request merged**